### ISSUE TITLE
Training on Other Datasets

### ISSUE THREAD

**Issue by Biyb** [juntaoJianggavin/RWKV-UNet#9]
Hello, author.
I noticed that besides the Synapse data and ACDC data, you have also tested on datasets such as BUSI, CVC-ClinicDB, CVC-ColonDB, Kvasir-SEG, ISIC2017, and GLAS. Since I plan to use your model as a comparative experiment, and my datasets are Kvasir, ClinicDB, ColonDB, CVC300, and ETIS. Among them, the two datasets, CVC300 and ETIS, have not been tested by you. Therefore, I want to use the code you provided to test the results. However, on the homepage of your code, I didn't find out how to debug for these datasets including BUSI, CVC-ClinicDB, CVC-ColonDB, Kvasir-SEG, ISIC2017, and GLAS. So I would like to consult you about the specific operation steps and the procedures for modifying the code.

**Comment by Mineuau123** [comment]
是的，希望作者大人可以提供一下在其他数据上训练和推理的代码

**Comment by SuChenHong** [comment]
Hello author! 
I have some questions. Why is the dice of ColonDB so high (including UNet and UNet++)? The data I saw in Polyper-PVT are relatively lower. I replicated it locally, and many model data can also match, but in many papers, the dice of ColonDB is very high. I've observed that the datasets are the same, is it because the method of calculating dice is different?

**Comment by juntaoJianggavin** [comment]
> Hello author! I have some questions. Why is the dice of ColonDB so high (including UNet and UNet++)? The data I saw in Polyper-PVT are relatively lower. I replicated it locally, and many model data can also match, but in many papers, the dice of ColonDB is very high. I've observed that the datasets are the same, is it because the method of calculating dice is different?

Hello, thanks for your question!
Many works on ColonDB use a 4:1 split and directly report the best model on the validation set, which can lead to higher Dice scores. In our setting, we use an 8:1:1 train/validation/test split: we select the best model on the validation set and then evaluate it on an independent test set, which is more rigorous and generally results in slightly lower but more reliable numbers. Also, different random seeds can change the data split and initialization, leading to noticeable variations in results.
